demo/README.md

```diff
@@ -0,0 +1,48 @@
+# Demo
```

Comment:
This document is greate. But we may move it to docs/ thus the readthedocs.io will also present this doc to authors.

Reply:
OK, I will move it to #Verifications-##Demo.

Reply:
Move the docs/demo.md.